google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/InsertDataAndQueryTable.java

```diff
@@ -79,20 +83,16 @@ public static void main(String... args) throws InterruptedException {
     }
 
     // Create a query request
-    QueryRequest queryRequest =
-        QueryRequest.newBuilder("SELECT * FROM my_dataset_id.my_table_id")
-            .setMaxWaitTime(60000L)
-            .setPageSize(1000L)
-            .build();
+    QueryJobConfiguration queryConfig =
+        QueryJobConfiguration.newBuilder("SELECT * FROM my_dataset_id.my_table_id").build();
```

Comment:
We should not be encouraging people to [use `select *` with BigQuery](https://stackoverflow.com/a/23991604).

Reply:
I did not know that. What should be used instead? Explicit list of column names?
I did not create that query, only modified the surrounding code, that is how that line is showed as "added". Leaving it as is.

Reply:
`SELECT my_column FROM my_dataset_id.my_table_id` would be appropriate.